### ISSUE TITLE
AlexNet代码问题

### ISSUE THREAD

**Issue by 1261017506** [chenyuntc/pytorch-book#145]
我在看AlexNet代码的时候发现程序中有两种写法，nn.Conv2d(3, 64, kernel_size=11, stride=4, padding=2)， nn.Conv2D(channels=96, kernel_size=11, strides=4, activation=‘relu‘),都是第一层卷积的，96这种方法一看就懂，但是为什么另一个是64
